org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/ReferenceUpdater.java

```diff
@@ -245,7 +245,7 @@ public void updateReference(ITextRegionDiffBuilder rewriter, IUpdatableReference
 		}
 		IScope scope = scopeProvider.getScope(updatable.getSourceEObject(), updatable.getEReference());
 		ISemanticRegion region = updatable.getReferenceRegion();
-		QualifiedName oldName = nameConverter.toQualifiedName(region.getText());
+		QualifiedName oldName = getQualifiedName(updatable);
```

Comment:
i am not sure if this changes semantics when there are two scope elements with the same qualified but different names (e.g. 1 imported, 1 not)

Reply:
Should not. It's more about getting qnames right (using default delimiter) :

`module "my.module" {} `

The old code would produce `"my` `module"`  qname , but should be `my` `module`